addons/crm/static/tests/mock_server/mock_model/crm_lead.js

```diff
@@ -0,0 +1,5 @@
+import { models } from "@web/../tests/web_test_helpers";
+
+export class CRMLead extends models.ServerModel {
+    _name = "crm.lead";
+}
```

Comment:
Ideally it should be imported from the parent module instead, in a similar way than we do with `mailModels`.

Reply:
@seb-odoo I have update the code, please have a look.